ooniapi/services/oonirun/src/oonirun/routers/v2.py

```diff
@@ -33,16 +37,26 @@
 def utcnow_seconds():
     return datetime.now(timezone.utc).replace(microsecond=0)
 
+class OonirunMeta(BaseModel):
+    run_type : str = Field(description="Run type", pattern="^(timed|manual)$")
+    is_charging : bool = Field(description="If the probe is charging")
+    probe_asn : str = Field(pattern=r"^([a-zA-Z0-9]+)$")
+    probe_cc : str = Field(description="Country code. Ex: VE")
+    network_type : str = Field(description="Ex: wifi")
+    website_category_codes : List[str] = Field(description="List of category codes that user has chosen to test (eg. NEWS,HUMR)", default=[])
+
+    def probe_asn_int(self) -> int:
+        return int(self.probe_asn.replace("AS", ""))
 
 class OONIRunLinkNettest(BaseModel):
     test_name: str = Field(
         default="", title="name of the ooni nettest", min_length=2, max_length=100
     )
-    inputs: List[str] = Field(
+    inputs: Optional[List[str]] = Field(
         default=[], title="list of input dictionaries for the nettest"
```

Comment:
I think `inputs` should be initialized to `None`, otherwise the validation for `targets_name` not being preset at the same time as `inputs` will fail.

Reply:
We can otherwise change this to be the empty list, but then we should change that also for `inputs_extra`